### PR TITLE
Fix FireFox High Contrast Mode for Radio Schedule

### LINKED ISSUE #3323: Radio Schedule: FireFox high contrast border bug
**Describe the bug**
Duration button is not displaying top border in FireFox high contrast mode:
Screenshot 2020-04-01 at 11 06 46

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/hausa?renderer_env=live in FireFox
2. Select High Contrast Mode
3. Scroll down to Radio Schedule section on a page

**Expected behaviour**
Top border is displayed in High Contrast Mode.

**Desktop (please complete the following information):**

- Browser: FireFox

**Testing notes**

- [x] This bug fix is expected to need manual testing.

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #3323

**Overall change:** 
Made duration border display in FireFox High Contrast Mode. 
The reason why `outline` is used instead of `border-top` is because during Radio Schedule accessibility swarm UX mentioned that there should be no extra border around the duration button (even if it's invisible) because it adds height.

Before:
Screenshot 2020-04-01 at 11 06 46

After:
Screenshot 2020-04-01 at 11 06 34

**Code changes:**

- Added `outline` to duration button.
- Updated snapshots.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-radio-schedule/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.1.0-alpha.28 | [PR#xxxx](https://github.com/bbc/psammead/pull/xxxx) Fix Duration border on FireFox High Contrast Mode |
```
Comment (OlgaLyubin) [inline packages/components/psammead-radio-schedule/CHANGELOG.md]:
```suggestion
| 0.1.0-alpha.28 | [PR#3324](https://github.com/bbc/psammead/pull/3324) Fix Duration border on FireFox High Contrast Mode |
```

**Review by Bopchy — DISMISSED** [review]
👍

**Review by Bopchy — COMMENTED** [review]
Checked this out in FF and it looks great, however it overrides the transparent border on the Button in IE. 

Screenshot 2020-04-01 at 18 07 03

Should confirm whether this is a major concern though

**Comment by OlgaLyubin** [comment]
Fixed so that IE high contrast mode looks like this:
Screenshot 2020-04-03 at 10 09 50

Still not ideal, but since IE is a low priority browser, it can stay like this.

**Review by Bopchy — APPROVED** [review]

**Review by hotinglok — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me on FF.
Screenshot 2020-04-03 at 16 00 40

Looks as described by Olga on IE11.
VirtualBox_MSEdge - Win10_03_04_2020_17_52_19

**Pull request merged**